500-fix-loading-spinner

SevLG opened the pull request:
https://scilifelab.atlassian.net/browse/TP-500

Feature: added new loading spinner based on shadcn component. Used with react "Suspense" on plot pages to show that the plot page components are loading, as they were a bit slow on first load of the page.

SevLG committed:
feature: added new loading spinner based on shadcn component. Used with react suspense on plot pages to show that the plot page components are loading

JanProgrammierung reviewed (commented):
Is the Loading component necessary since it just has a function calling the SpinnerWithText component?
@SevLG

SevLG committed:
fix: removed unnecessary extra component SpinnerWithText, moved content to Loading.tsx

SevLG commented:
> Is the Loading component necessary since it just has a function calling the SpinnerWithText component? @SevLG

Moved content of SpinnerWithText to Loading and removed SpinnerWithText
